src/einsteinpy/utils/christoffel.py

```diff
@@ -80,6 +80,36 @@ def christoffels(list2d, syms):
     return christlist
 
 
+def simplify_tensor_helper(v):
```

Comment:
Make this private!

Reply:
Also, the parameter, should be more than `v`

Reply:
> Also, the parameter, should be more than `v`

actually the parameter can be both sympy expression or list. So better keep it ```variable``` or ```v```